From 29f76bf38c933eba106ca7928e03f094365ae31c Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@lucaweiss.eu>
Date: Thu, 3 Apr 2025 18:10:57 +0200
Subject: [PATCH 2/2] Revert "tests/common: workaround CMake dependency bug"

  CMake Error at tests/common/CMakeLists.txt:60 (add_dependencies):
    The dependency target "GMock" of target "lomiri-download-manager-test-lib"
    does not exist.

  CMake Error in tests/common/CMakeLists.txt:
    The dependency target "GMock" of target
    "lomiri-download-manager-test-lib_autogen_timestamp_deps" does not exist.

This reverts commit b9739cdefcd47229b411d7c7e0b81699854aa0bb.
---
 tests/common/CMakeLists.txt | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/tests/common/CMakeLists.txt b/tests/common/CMakeLists.txt
index 5409613f..8cd1b1c7 100644
--- a/tests/common/CMakeLists.txt
+++ b/tests/common/CMakeLists.txt
@@ -51,10 +51,3 @@ target_link_libraries(${TARGET}
     lomiri-download-manager-common${QT_SUFFIX}
     lomiri-download-manager-client${QT_SUFFIX}
 )
-
-# Workaround CMake bug [1]. This is actually worked around in cmake-extras
-# already [2], but don't know why it doesn't work here.
-# [1] https://gitlab.kitware.com/cmake/cmake/-/issues/25766
-# [2] https://gitlab.com/ubports/development/core/cmake-extras/-/merge_requests/18
-#     https://gitlab.com/ubports/development/core/cmake-extras/-/merge_requests/19
-add_dependencies(${TARGET} GMock)
-- 
2.49.0

